src/textlint-rule-unexpanded-acronym.js

```diff
@@ -67,18 +67,29 @@ export default function (context, options = {}) {
                 } else if (!includes(acronymJoiningWords, word) // ignore of and...
                     && acronymCreator.canExtractAcronym()) {
                     // Create Acronym
-                    var acronym = acronymCreator.extractAcronym();
-                    if (isWordSatisfy(acronym)) {
-                        expandedAcronymList.push(acronym);
+                    var acronyms = acronymCreator.extractAcronym();
+                    if(!Array.isArray(acronyms)){
+                        if (isWordSatisfy(acronyms)) {
+                            expandedAcronymList.push(acronyms);
+                        }
+                    }else{
+                        Array.from(acronyms).forEach(acronym => {
```

Comment:
`acronyms` is array, because `if(!Array.isArray(acronyms)){ ... } else {`.
Not need `Array.from`

Reply:
yep, I should change this